keen/client.py

```diff
@@ -229,16 +229,20 @@ def count(self, event_collection, timeframe=None, timezone=None, interval=None,
         example: [{"property_name":"device", "operator":"eq", "property_value":"iPhone"}]
         :param group_by: string or array of strings, the name(s) of the properties you would
         like to group your results by.  example: "customer.id" or ["browser","operating_system"]
+        :param order_by: dictionary object containing the property_name to order by and the
```

Comment:
Here and everywhere `order_by` is a param, I'd generally prefer to see a `Direction.DESC` enum constant (and `Direction.ASC` of course) so there's no room for typos and the code is self-documenting. Even just typing this comment I had to scroll up and check if it was `ASC` or some other substring of "ascending" which isn't great. I know enums don't exist in Python until 3.4, so that would imply pulling in the backport which is python package `enum34` or something like that, but maybe it's worth it. Not sure if the `six` package does anything about automatically normalizing enums, but we already pull in `six` so maybe look at that? Another way to do it would be to just follow something like the typesafe enum pattern or just have string constants in a `Direction` class or anything else that makes this easier to use and less error-prone.

Reply:
Good point. While we're here, we might as well make this situation better. I'll probably just have a Direction class and use that.